ReactiveCocoa/Swift/Property.swift

```diff
@@ -97,6 +109,16 @@ public final class MutableProperty<Value>: MutablePropertyType {
 		}
 	}
 
+	/// A signal that will send the property's changes over time,
+	/// then complete when the property has deinitialized.
+	public lazy var signal: Signal<Value, NoError> = { [unowned self] in
+		var extractedSignal: Signal<Value, NoError>?
```

Comment:
It needn't be an IUO (https://github.com/andersio/ReactiveCocoa/commit/2651f6ff98ac77bcf84bd913680ca5da17076a2d#diff-84e9bb9c50af433fe0ae0e9e9bc2469fR129), but I think it makes sense to extract the signal only when it is needed. What do you think?